backend/src/src-licenses/src/main/java/com/siemens/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -370,7 +372,10 @@ public RequestStatus updateLicense(License inputLicense, User user, User request
 
             License dbLicense = new License();
             if(inputLicense.isSetId()) {
-                dbLicense = licenseRepository.get(inputLicense.getId());
+                License dbLicenseTmp = licenseRepository.get(inputLicense.getId());
```

Comment:
to me, it seems to work fine with the old line. I'm not convinced that any change is necessary here. Do you know of specific steps to reproduce whatever problem this is solving?